dynamo/plot/scatters.py

```diff
@@ -260,6 +267,23 @@ def scatters(
         marker: `str` (default: None)
             The marker style. marker can be either an instance of the class or the text shorthand for a particular
             marker. See matplotlib.markers for more information about marker styles.
+        affine_transform_degree:
+            Transform coordinates of points according to some degree.
+        affine_transform_A:
+            Coefficients in affine transformation Ax + b. 2D for now.
+        affine_transform_b:
+            Bias in affine transformation Ax + b.
+        stack_colors:
+            Whether to stack all color on the same ax passed above.
+            Currently only support 18 sequential matplotlib default cmaps assigning to different color groups.
+            (#colors should be smaller than 18, reuse if #colors > 18. To-do: generate cmaps according to #colors)
+        stack_colors_threshold:
+            A threshold for filtering points values < threshold when drawing each color.
```

Comment:
for cells with threshold less than stack_colors_threshold, can you set the color to them to be lightblue instead of removing those cells?